tests/openapi/checks/resource/v2/test_GlobalSchemeDefineHTTP.py

```diff
@@ -0,0 +1,43 @@
+import os
```

Comment:
We can maybe improve all the test files to be in this format:
https://github.com/bridgecrewio/checkov/blob/1d4fac764c3b7d1e27b9d1b2004cd7976077b9b4/tests/github/checks/test_python_policies.py
This is something Anton made for github and allows each test to be added as a single liner, instead the whole file.

Reply:
Great, done